### DISCUSSION TITLE
Error in resizeDynamic images

### DISCUSSION THREAD

**Discussion by ahhaweb99** [flyntwp/flynt#436] [Q&A] [answered]
Hi,

I've been using the resizeDynamic function of flynt succesfully for a while now until recently when for some images i am receiving this error in the img src output:

`<img class="figure-image lazyloaded" src="<br />
<b>Warning</b>:  Undefined array key " dirname"="" in="" <b="">`

Error is coming from:
`/lib/Utils/TimberDynamicResize.php on line 149`

Any idea why this would start happening?

Any help appreciated.

Thanks

**Comment by koukouchris** [comment]
Hi @ahhaweb99 
That error could be from invalid image url,
Can you share the twig code of the img element? 
Is the source of the image correct? 

you can use [dump function](https://twig.symfony.com/doc/2.x/functions/dump.html) inside twig to see if its a valid url, example: 
```
<pre>
{{ dump(image.src) }}
</pre>
```
if its `NULL` then the image source is not valid. Let me know if this helps:)

**Comment by aaronmeder** [comment, marked as answer]
It looks like your source isn't valid, as @koukouchris stated.

Sometimes it might help to just before the image output do a 
`{% set image = Image(image) %} `
to make sure you have a Timber\Image.

**Reply by ahhaweb99** [reply]
Turns out this was because the image didnt exist in the post thumbnail. So i added a check to see if image exists first. It's a little strange because this didnt happen previously, even though i had posts without thumbnails. Anyway, i've solved simpy by adding:

`{% if item.image %}`

...before accessing the image.

Thanks for the replies.
